Lib/test/test_typing.py

```diff
@@ -1095,14 +1073,7 @@ class Node(Generic[T]): ...
                   Union['T', int], List['T'], typing.Mapping['T', int]]
         for t in things + [Any]:
             self.assertEqual(t, copy(t))
-            self.assertEqual(t, deepcopy(t))
-            if sys.version_info >= (3, 3):
-                # From copy module documentation:
-                # It does "copy" functions and classes (shallow and deeply), by returning
-                # the original object unchanged; this is compatible with the way these
-                # are treated by the pickle module.
-                self.assertTrue(t is copy(t))
-                self.assertTrue(t is deepcopy(t))
+            self.assertEqual(repr(t), repr(deepcopy(t))) # Use repr() because of TypeVars
```

Comment:
Doesn't copying classes preserve identity?

Reply:
``deepcopy`` creates new ``TypeVar`` instances, and they always compare unequal. There is already an issue about this on typing tracker. I will fix it there soon and update this PR.